doc/developers-guide/plugin-development/a-day-in-the-life-of-a-plugin.md

```diff
@@ -38,55 +38,63 @@ The `getmanifest` method is required for all plugins and will be called on start
 
 ```json
 {
-  "options": [
-    {
-      "name": "greeting",
-      "type": "string",
-      "default": "World",
-      "description": "What name should I call you?",
-      "deprecated": false,
-      "dynamic": false
-    }
-  ],
-  "rpcmethods": [
-    {
-      "name": "hello",
-      "usage": "[name]",
-      "description": "Returns a personalized greeting for {greeting} (set via options)."
-    },
-    {
-      "name": "gettime",
-      "usage": "",
-      "description": "Returns the current time in {timezone}",
-      "long_description": "Returns the current time in the timezone that is given as the only parameter.\nThis description may be quite long and is allowed to span multiple lines.",
-      "deprecated": false
-    }
-  ],
-  "subscriptions": [
-    "deprecated_oneshot",
-    "connect",
-    "disconnect"
-  ],
-  "hooks": [
-    { "name": "openchannel", "before": ["another_plugin"] },
-    { "name": "htlc_accepted" }
-  ],
-  "featurebits": {
-    "node": "D0000000",
-    "channel": "D0000000",
-    "init": "0E000000",
-    "invoice": "00AD0000"
-  },
-  "notifications": [
-    {
-        "method": "mycustomnotification"
-    }
-  ],
-  "custommessages": [
-    11008, 11010
-  ],
-  "nonnumericids": true,
-  "dynamic": true
+	"options": [
+		{
+			"name": "greeting",
+			"type": "string",
+			"default": "World",
+			"description": "What name should I call you?",
+			"deprecated": false,
+			"dynamic": false
+		}
+	],
+	"rpcmethods": [
+		{
+			"name": "hello",
+			"usage": "[name]",
+			"description": "Returns a personalized greeting for {greeting} (set via options)."
+		},
+		{
+			"name": "gettime",
+			"usage": "",
+			"description": "Returns the current time in {timezone}",
+			"long_description": "Returns the current time in the timezone that is given as the only parameter.\nThis description may be quite long and is allowed to span multiple lines.",
+			"deprecated": false
+		}
+	],
+	"subscriptions": [
+		"deprecated_oneshot",
+		"connect",
+		"disconnect"
+	],
+	"hooks": [
+		{
+			"name": "openchannel",
+			"before": [
+				"another_plugin"
+			]
+		},
+		{
+			"name": "htlc_accepted"
+		}
+	],
+	"featurebits": {
+		"node": "D0000000",
+		"channel": "D0000000",
+		"init": "0E000000",
+		"invoice": "00AD0000"
+	},
+	"notifications": [
+		{
+			"method": "mycustomnotification"
+		}
+	],
+	"custommessages": [
+		11008,
+		11010
+	],
+	"nonnumericids": true,
+	"dynamic": true
```

Comment:
Formatting here is weird, maybe pass through jq?

Reply:
Done.